### ISSUE TITLE
Feature Add: Add mTLS support

### ISSUE THREAD

**Issue by ChrisMcGowan** [cloudfoundry/windows-syslog-release#17]
In order to have parity with the linux syslog release, please add support for mTLS to this windows release as it's configured for the linux version.

This is support a mixed diego-cell environment (linux and windows) that utilize syslog and require mTLS.

Thank you

**Comment by Benjamintf1** [comment]
Just saw this now. Just a heads up, might want to reach out in the logging and metrics slack if you want people to see this issue. Also, ping me in slack if you make a pr for this, or if you want any advice on making a pr. 

Main tests are here https://github.com/cloudfoundry/windows-syslog-release/tree/main/tests, [please introduce a new test case like syslog-release if you do](https://github.com/cloudfoundry/syslog-release/blob/main/tests/manifests/tls-forwarding-mtls.yml), https://github.com/cloudfoundry/syslog-release/blob/49b260be5fe5274e9b9c49c145e5e08095e5528f/tests/acceptance_test.go#L236. Probably also requires a pr to blackbox to enable optional mtls as well.

**Comment by MirzaAli-Baig** [comment]
Did we resolve this. Any timelines to it

**Comment by Benjamintf1** [comment]
No, I don't see a pr in https://github.com/cloudfoundry/blackbox/pulls. If you would like to submit a pr to make this work, feel free to ping me in the https://cloudfoundry.slack.com/archives/CUW93AF3M cloudfoundry slack in #logging-and-metrics and let me know that you have a pr enabling mtls for windows-syslog-release(/blackbox) and I can probubly give you some pretty quick feedback/possibly merging.

**Comment by MirzaAli-Baig** [comment]
I am administrator. Can you please fix it and provide release.
